doc/api/magic-plot.md

```diff
@@ -155,12 +161,20 @@ When plotting a histogram, it divides a range with the number of bins - 1 to cal
 
 ### Specifying breaks
 
-Breaks allow you to set custom intervals for a histogram. You can specify breaks by passing desired each end and break points separated by whitespace after `-B/--breaks`. Since those break points define a range of data points to plot, bar width, and number of bars in a histogram, make sure to pass more than 1 point that is strictly increasing and includes at least one data point. Note that using both `-b/--bins` and `-B/--breaks` isn't allowed. 
+Breaks allow you to set custom intervals for a histogram. You can specify breaks by passing desired each end and break points separated by whitespace after `-B/--breaks`. Since those break points define a range of data points to plot, bar width, and number of bars in a histogram, make sure to pass more than 1 point that is strictly increasing and includes at least one data point.
 
 ```{code-cell} ipython3
 %sqlplot histogram --table penguins.csv --column body_mass_g --breaks 3200 3400 3600 3800 4000 4200 4400 4600 4800
 ```
 
+### Specifying binwidth
+
+Binwidth allows you to set the width of bins in a histogram. To specify the binwidth, pass a desired width after `-W/--binwidth`. Since the binwidth determines details of distribution, make sure to pass a suitable positive numeric value based on your data.
```

Comment:
I think we should also add when to use each of bins/ breaks/ binwidth (in which scenario each is helpful)?

Reply:
Information about when each parameter is useful: [bins](https://github.com/ploomber/jupysql/pull/789/files#diff-d2bac2aa48fbf4cb4aee34358d99fb9653fe443d5d3359ea2848dd36bdaa2f3bR156-R158), [breaks](https://github.com/ploomber/jupysql/pull/789/files#diff-d2bac2aa48fbf4cb4aee34358d99fb9653fe443d5d3359ea2848dd36bdaa2f3bR166), [binwidth](https://github.com/ploomber/jupysql/pull/789/files#diff-d2bac2aa48fbf4cb4aee34358d99fb9653fe443d5d3359ea2848dd36bdaa2f3bR174)